syntax = "proto3";

package dev_observer.api.types.processing;

import "google/protobuf/timestamp.proto";

message ProcessingItem {
  string id = 1;

  optional google.protobuf.Timestamp next_processing = 2;
  optional google.protobuf.Timestamp last_processed = 3;
  optional string last_error = 4;
  bool no_processing = 5;

  oneof entity {
    string github_repo_id = 100;
  }
}